### DISCUSSION TITLE
Tasmota password

### DISCUSSION THREAD

**Discussion by kted** [UnchartedBull/OctoDash#1825] [General]
So, how do I enter the required useraname & password for my Sonoff basic running Tasmota? I only see IP and INDEX in settings.
So far, I've been using it with curl, but now that it's supported, I want to know how to enter my credentials.

**Comment by jneilliii** [comment]
Tasmota actions are handled by my Tasmota plugin, all that other info is configured in that plugin's settings, not OctoDash.
